projects/react/package.json

```diff
@@ -32,7 +32,7 @@
     "devDependencies": {
         "@testing-library/react": "13.4.0",
         "@testing-library/react-hooks": "8.0.1",
-        "@testing-library/user-event": "14.4.3",
+        "@testing-library/user-event": "14.5.1",
```

Comment:
See
* https://github.com/testing-library/user-event/issues/1161

Reply:
Need upgrade to Typescript 4.5.0+